src/messaging/ReliableMessageMgr.cpp

```diff
@@ -471,6 +465,26 @@ CHIP_ERROR ReliableMessageMgr::MapSendError(CHIP_ERROR error, uint16_t exchangeI
     return error;
 }
 
+void ReliableMessageMgr::CalculateNextRetransTime(RetransTableEntry *& entry)
```

Comment:
Why is this reference-to-pointer?  We are not mutating `entry` in here, right?  I'd expect it to just be a pointer...

Reply:
@tcarmelveilleux suggested this since it guarantees the pointer cannot be null.

Reply:
No, it doesn't.  What I assume @tcarmelveilleux meant was `CalculateNextRetransTime(RetransTableEntry & entry)`

Reply:
That in fact makes more sense...